modules/installation-osp-enabling-kuryr.adoc

```diff
@@ -9,16 +9,16 @@ You can enable Kuryr in your project by editing a parameter in the `install-conf
 
 .Prerequisites
 
-* Kuryr requires greater than the minimum recommended resources. As an administrator, increase Red Hat OpenStack Platform's (RHOSP) resource quotas:
+* Kuryr requires greater than the minimum recommended resources. As an administrator, increase Red Hat OpenStack Platform's ({rh-openstack}) resource quotas:
```

Comment:
should use {rh-openstack-first} instead of replacing just the RHOSP part with {rh-openstack}.

Oh I see why, because of the "'s". Maybe reword to work around it? "increase the resource quotas for {rh-openstack-first}" ?

Reply:
Oh, whoops. Yeah. Fixed! Thanks.